nats-base-client/kv.ts

```diff
@@ -212,10 +221,14 @@ export class Bucket implements KV, KvRemove {
   }
 
   subjectForBucket(): string {
-    return `${kvSubjectPrefix}.${this.bucket}.>`;
+    return `${this.subjPrefix}.${this.bucket}.>`;
   }
 
   subjectForKey(k: string): string {
+    return `${this.subjPrefix}.${this.bucket}.${k}`;
+  }
+
+  getSubjectForKey(k: string): string {
```

Comment:
maybe pick a different name that calls out the difference.
`get` is a bit non descript.